### DISCUSSION TITLE
Usage of `std_types` in `create_pipe`

### DISCUSSION THREAD

**Discussion by jkisse** [e2nIEE/pandapipes#523] [General]
Currently, the default `create_pipe` uses standard pipe types. The only value from the standard type library that is relevant to the pipeflow calculation is the diameter. (The main reason behind these standard types is the structure of pandapower - for power lines, many more parameters are read from the standard type library).
To set the diameter individually, the `create_pipe_from_parameters` function has to be called. I have the impression that this one is much more relevant than "create using std_types".

I'd like to discuss whether the default `create_pipe` function should rather be the one that is today `create_pipe_from_parameters`. It shouldn't be too complicated to pass everything to a `create_pipe_from_std_types` function, if a `std_type=...` is being passed, so it could be backward-compatible.

What do you think about the usage of standard types and the default create functions?

**Comment by dlohmeier** [comment]
I think, that's a great idea!

**Comment by dlohmeier** [comment]
I think, what we would also need is that the `create_pipe_from_parameters` function remains in place with a DeprecationWarning for some time then.
